tests/storage-spec.js

```diff
@@ -149,13 +149,13 @@ describe('languageStorage factory', function () {
 
             this.languageChoices = [
                 {
-                    id: 'en_us',
-                    label: 'English',
-                },
-                {
                     id: 'pl',
                     label: 'Polski',
                 },
+                {
+                    id: 'en_us',
+                    label: 'English',
+                },
```

Comment:
Can you include a comment somewhere, maybe in a brief at the top of this file or just this suite, the reasoning to ensure the tests don't assert against the default choice?